src/OsmMom.sol

```diff
@@ -73,25 +72,37 @@ contract OsmMom {
         }
     }
 
-    mapping (bytes32 => address) public osms;
-
-    constructor() public {
+    constructor(address vat_) {
+        vat = vat_;
         owner = msg.sender;
+        emit SetOwner(msg.sender);
     }
 
-    function setOsm(bytes32 ilk, address osm) external note onlyOwner {
-        osms[ilk] = osm;
-    }
-
-    function setOwner(address owner_) external note onlyOwner {
+    function setOwner(address owner_) external onlyOwner {
         owner = owner_;
+        emit SetOwner(owner_);
```

Comment:
`owner` should be logged here, following `ds-auth` pattern
```suggestion
        emit SetOwner(owner);
```

Reply:
actually, it should be fine as more gas efficient as followed in latest code as well.